tools/infrasys/src/main.rs

```diff
@@ -15,7 +15,7 @@ use std::collections::HashMap;
 use std::num::NonZeroUsize;
 use std::path::{Path, PathBuf};
 use std::{fs, process};
-use structopt::StructOpt;
```

Comment:
Does it not build if we don't `use clap`?

We should probably remove `structopt` in favor of just `clap` going forward, though it could be done as a separate task. I haven't investigated what a migration looks like, but the feature set we're using should have something synonymous in `clap`.

Check out the structop docs here: https://docs.rs/structopt/latest/structopt/#maintenance

Reply:
From what I can tell we would want to switch to `clap` sooner rather than later. Because of the version bump, our tools don't build unless we prefix every `clap` with `structopt::`, or we add it to the use declaration.

Reply:
> it could be done as a separate task

Yes, I think replacing StructOpt with `clap` should be done separately from regular library updates.

> Does it not build if we don't `use clap`

Seems like this change is fine... it makes it compile!